packages/react-components/react-slider/src/components/Slider/Slider.types.ts

```diff
@@ -104,10 +107,8 @@ export type SliderOnChangeData = {
   value: number;
 };
 
-export type SliderProps = Omit<
-  ComponentProps<Partial<SliderSlots>, 'input'>,
-  'defaultValue' | 'onChange' | 'size' | 'value'
-> &
-  SliderCommons;
-
-export type SliderState = ComponentState<SliderSlots> & SliderCommons;
+export type SliderState = ComponentState<SliderSlots> &
+  Pick<
+    SliderProps,
+    'value' | 'defaultValue' | 'disabled' | 'max' | 'min' | 'origin' | 'size' | 'step' | 'vertical' | 'onChange'
```

Comment:
same alphabetical ordering thing here.

But also -- do we need all these on state? Looking at Slider, it seems that the only values actually used on `state` (aside from slots) are `disabled`, `size`, and `vertical`.